content/pt-br/docs/concepts/cluster-administration/proxies.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Proxies no Kubernetes
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+Esta página descreve o uso de proxies com Kubernetes. 
+
+
+<!-- body -->
+
+## Proxies
+Existem vários tipos diferentes de proxies que você pode encontrar usando Kubernetes:
+
+
+1.  O [kubectl proxy](/docs/tasks/access-application-cluster/access-cluster/#directly-accessing-the-rest-api):
+
+    - executa na máquina do usuário ou em um pod
+    - proxies de um host local endereçam para o servidor de API do kubernetes
+    - cliente para o proxy usa HTTP 
+    - proxy para o servidor de API usa HTTPS 
+    - localiza o servidor de API
+    - adiciona os cabeçalhos de comunicação.
+
+1.  O [apiserver proxy](/docs/tasks/access-application-cluster/access-cluster/#discovering-builtin-services):
+
+    - é um bastion server, construído no servidor de API
+    - conecta um usuário fora do cluster com os IPs do cluster que não podem ser acessados de outra forma
+    - executa dentro do processo do servidor de API
+    - cliente para proxy usa HTTPS (ou http se o servidor de API for configurado)
```

Comment:
s/http/HTTP

Reply:
Please, look into the new changes and see if we can keep them.